docs/api-docs/getting-started/rest-api-authentication.md

```diff
@@ -1,260 +1,210 @@
 
-### App API credentials
+**Guard these values closely.** The `client_id` and `client_secret` will never change; `access_token`s do not expire based on time and cannot be manually invalidated. It's best practice to limit each account's [OAuth scope](#oauth-scopes) to only the privileges needed to complete that app or user's designated tasks. Create separate API accounts for each app, store API user, and/or function.
 
-Developers can create app API credentials in the BigCommerce [Developer Portal](https://devtools.bigcommerce.com). App API credentials are used during the OAuth flow to request authorization “on behalf” of a store owner, allowing the app to make API requests against store data. App API credentials are OAuth only, and the store owner must install the app before the app is granted access to the store.
-
-## Obtaining store API credentials
-
-The following steps outline how to generate store API Credentials.
-
-1. Navigate to **Advanced Settings** > **API Accounts** > **Create API Account**.
-2. Give the account a name (it will only be visible to store users).
-3. In the **OAuth Scopes** section, select the minimum scopes the app will require.
-4. Select **Save**.
-
-A successful save will display a pop-up containing an Access Token. Your app will need this credential to run authenticated requests. A `.txt` file containing the same credentials will (on most browsers) automatically download to your computer. This file also contains the base API Path for your store, preconfigured for the V3 API.
-
-The base API path will look something like this: 
+<!-- theme: info -->
+> Although the **client ID** value uniquely identifies the app or user making a request, you don't need to pass it in the header of each API request.
 
-`https://api.bigcommerce.com/stores/123456/`. 
+## Store API accounts
 
-In the base path, the store hash is the `123456`. You will use this to make API requests.
+Merchants generate single-store API credentials when they create API accounts in their store control panel (**Advanced Settings** > **API Accounts**). Use these credentials to read and change one store's data with BigCommerce's APIs. Store API accounts' access tokens and OAuth scopes can't be changed. 
 
-To get started making requests, see [API Requests](/api-docs/getting-started/basics/making-requests).
+In addition to the [API account components](#api-accounts) in the preceding section, store API accounts contain the following attributes out of the box:
+* an `access_token`, which accompanies most REST API requests;
+* the **client name** is a label for your convenience, and it doesn't accompany requests;
+* the **API path** is the URL to which you make requests. The API path won't change, but it will have `/v3/` or `/v2/` appended to it, depending on which version is current for the endpoint you're querying.
```

Comment:
```suggestion
* the **API path** is the URL to which you make requests. The API path won't change, but it will have `/v3/` or `/v2/` appended to it, depending on the current version for the endpoint you're querying.
```